charts/argo-cd/values.yaml

```diff
@@ -115,6 +115,13 @@ repoServer:
     #   cpu: 10m
     #   memory: 64Mi
 
+# -- ApplicationSet Controller
+applicationsetController:
+  image:
+    repository: quay.io/argoproj/argocd-applicationset # defaults to global.image.repository
```

Comment:
This needs to be on akuity Quay repo https://github.com/akuityio/akuity-enterprise/issues/18

Reply:
Should we change this in a separate PR?

Reply:
I think so. I left this as a separate issue for discussion https://github.com/akuityio/akuity-enterprise/issues/18.